### DISCUSSION TITLE
Inline comments hidden in "files changed" tab

### DISCUSSION THREAD

**Discussion by ts46236** [community/community#64122] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I just noticed that now github completely hides inline comments.  I have to go to the conversations link drop down at the top of the 'files changed' view and go to each one by one. 

There are 11 comments...11 times i have to:
1. scroll to the top of page and open the conversations drop down
2. expand resolved conversations
3. click one (have to scroll in this list and remember which one I chose last time to know which to choose now)

I have to repeat that 11 times. If i have 30 comments then multiply that by 3. This is ridiculously slow and frustrating

How can I get comments to show inline????

**Comment by vacisdev** [comment]
You can view all inline comments by clicking on the 'Files changed' tab and selecting the 'Show comments' option in the review dropdown menu, or by using the 'n' and 'p' keyboard shortcuts to navigate through the comments.

**Reply by ts46236** [reply]
@Jisll thank you, but there is no show comments option that I see in this modal:

![image](https://github.com/community/community/assets/88683629/1c8e5a95-74f7-4987-92f0-a4180cf2e6a9)

**Reply by ts46236** [reply]
![image](https://github.com/community/community/assets/88683629/59965161-8002-4be9-8634-78ca0b906c75)

The show comments is selected but the comments are still invisible in the page

**Comment by rainbow1016** [comment]
very good question!
